static/sass/styles.scss

```diff
@@ -352,3 +352,31 @@ h3 {
 .p-checkbox {
   position: relative;
 }
+
+.p-strip--charmhub-light {
```

Comment:
Could you move these styles to a strip SCSS file. Styles here are for bugs and hacks that are to be removed.

Maybe `.p-strip--suru-light` or extend the existing strip with `.p-strip--light.has-suru`?